Mathlib/Algebra/MvPolynomial/Rename.lean

```diff
@@ -70,15 +70,13 @@ lemma map_comp_rename (f : R →+* S) (g : σ → τ) :
 
 @[simp]
 theorem rename_rename (f : σ → τ) (g : τ → α) (p : MvPolynomial σ R) :
-    rename g (rename f p) = rename (g ∘ f) p :=
-  show rename g (eval₂ C (X ∘ f) p) = _ by
-    simp only [rename, aeval_eq_eval₂Hom]
-    -- Porting note: the Lean 3 proof of this was very fragile and included a nonterminal `simp`.
-    -- Hopefully this is less prone to breaking
-    rw [eval₂_comp_left (eval₂Hom (algebraMap R (MvPolynomial α R)) (X ∘ g)) C (X ∘ f) p]
-    simp only [comp_def, eval₂Hom_X']
-    refine eval₂Hom_congr ?_ rfl rfl
-    ext1; simp only [comp_apply, RingHom.coe_comp, eval₂Hom_C]
+    rename g (rename f p) = rename (g ∘ f) p := by
```

Comment:
Let me highlight this change - to get another pair of eyes that the porting note is likely not actionable.